docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -31,13 +28,34 @@ Run the following command:
   ```
 
 #### **Kyma Dashboard**
-1. Navigate to `Cluster Details`.
-2. Open the Istio module's configuration.
-3. Click **Edit**.
-4. Add the number of trusted proxies and click **Update**.
-   ![Add the numTrustedProxies](./assets/01-00-num-trusted-proxies-ui.svg)
+1. Navigate to Cluster Details and select Modify Modules.
+2. Choose the Istio module and select Edit.
+3. In the General section, add the number of trusted proxies.
+4. Select Save.
 <!-- tabs:end -->
 
+### Configure the externalTrafficPolicy in the Istio Custom Resource (GCP and Azure only)
```

Comment:
```suggestion
### Configure the External Traffic Policy in the Istio Custom Resource (GCP and Azure Only)
```

Reply:
The k8s field name is [externalTrafficPolicy](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip), as so I would not like it to be separated

Reply:
OK, now I see that't a busola tutorial :D
Maybe then we could use the full name here as well (Gateway external traffic policy)?

Reply:
There are Busola and kubectl instructions, but I agree that Gateway External Traffic Policy would be more universal